test/integration/scenarios/network/peers_blacklist.js

```diff
@@ -76,21 +67,25 @@ module.exports = params => {
 				);
 			});
 
-
 			describe('when a node blacklists an ip', () => {
 				before(done => {
+					if (!('access' in params.configurations[0].peers)) {
```

Comment:
What is that? Please add comments.

Reply:
It looks straightforward to me. The new config files don't have all the parameters in place. So, if they don't exist we need to add them. The PR with the new config files was merged. I assume this is expected after it.

Reply:
Are you sure? I think default config contains all the parameters, network-specific configs only overwriting those that are different from default ones.

Reply:
Well, the test has failed because of it. Check in here https://github.com/LiskHQ/lisk/blob/1.2.0/config/devnet/config.json .  There is not `syncing` parameter.

Reply:
https://github.com/LiskHQ/lisk/blob/3616b1f45bcec048dbf9333d363c290dc10e71b2/config/default/config.json#L89

Reply:
That default doesn't work as you think. The test suite gets `config/devnet/config.json` which doesn't have `peers.access` parameter, neither `peers.access.blackList`. Consequently it fails if we don't pass the control. Perhaps we have a problem in the new mechanism to get the configs...